source/types/FormTypes.ts

```diff
@@ -73,6 +73,7 @@ export interface Action {
   color?: string;
   hasCondition?: boolean;
   conditionalOn?: string;
+  message?: string;
```

Comment:
Very minor, but this name might be a little too generic? Maybe something that tells me that this has to do with bankId would be better.

Reply:
changed the message attribute to signMessage